projects/ResxWithHtmlContent/Messages.resx

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+   <resheader name="reader">
+    <value>System.Resources.ResXResourceReader</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter</value>
+  </resheader>
+  <data name="Html" xml:space="preserve">
```

Comment:
Shouldn't this node and the two nodes below in separate projects so you can show one 'fails' and the other 'succeeds'. The CompliantCSharp package doesn't have html content.

Reply:
I would say not. The fact that is reports on the non-compliant, and not an the escaped compliant nodes seems fine to me.